src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -206,26 +213,13 @@ def get_display(my):
             'type': 'mouseout',
             'bgcolor': bgcolor,
             'cbjs_action': '''
-            bvr.src_el.setStyle("background", bvr.bgcolor)
+            bvr.src_el.setStyle("border", "3px dashed #b8e5f5")
```

Comment:
we have different themes.. so it has to be based on the bgcolor or a intensity variation of it.. like you can see it up there

bgcolor = div.get_color("background3")
 bgcolor2 = div.get_color("background3", -3)
or

         bgcolor2 = div.get_color("background3", (-3, 10, 20))